index.html

```diff
@@ -9,37 +9,160 @@
 </head>
 <body>
     <div class="wrapper">
-        <header>
-          <nav>
-              <a href="index.html" class="logo">Charles</a>
-              <img src="images/Union.png" alt="MenuBar">
-          </nav>
-        </header>
-        <section class="content">
-            <article>               
-                <h1>I'm Charles Nice to meet you!</h1>
-                <p>
-                    My Name is Okoye Charles and I’m a software developer!
-                     I can help you build a product ,
-                    feature or website Look through some of my work and experience! 
-                    If you like what you see and need help on a project, I am available
-                     for hire.
-                </p>
-            </article>
-        
-            <div class="social-header">
-                <p>LET'S CONNECT</p>
-            </div>
-            <nav class="social-links">
-                <ul>
-                    <li><a href="https://twitter.com/OkoyeCharles_"><img src="images/Vector.png" alt="Twitter Icon"></a></li>
+      <div class="home-wrap">
+        <div class="first-section">
+          <header>
```

Comment:
- [ ] [REQUIRED] Please do this, the container of the section should be a `section` tag not a `div`.

Reply:
@mukhammadrizooff im done